app/javascript/components/admin/site_settings/Settings.jsx

```diff
@@ -1,7 +1,37 @@
 import React from 'react';
+import { Stack } from 'react-bootstrap';
+import useSiteSettings from '../../../hooks/queries/admin/site_settings/useSiteSettings';
+import Spinner from '../../shared/stylings/Spinner';
+import useUpdateSiteSetting from '../../../hooks/mutations/admins/site_settings/useUpdateSiteSetting';
+
+export default function Settings() {
+  const { isLoading, data: siteSettings } = useSiteSettings();
+  const updateSiteSetting = useUpdateSiteSetting();
```

Comment:
Isn't the `useUpdateSiteSetting` mutation expects an `id` argument?
There's no need (may be better to avoid) to pass it manually in the request body the as a `SettingId`.

Reply:
Yup good catch, I knew something was weird with this :laughing:

Reply:
Actually I remembered why I did this, it is because on the Settings.jsx page, there are multiple settings and different id's, so I can not just pass one id on line 9 since it is not known which id is being updated

Reply:
That being said though, I should remove the id param in `useUpdateSiteSetting`